docs/client-connectors/databricks-client/read-from-collection.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 40
+title: Read from a Collection
+---
+
+The Macrometa Collections Databricks Connector allows you to integrate [Apache Spark](https://spark.apache.org/) with [Macrometa](https://www.macrometa.com/docs/collections/) collections, allows you to read data from Macrometa collections using Apache Spark.
+
+1. Set up your source options:
```

Comment:
Where do I set up these source and target options? I get that there is a text file...somewhere...but I have no idea where I enter these parameters. Do we need to call that out, or will people using DataBricks just know automatically?

Reply:
@diana-macrometa - Typically, Databricks users will compose this code within the Databricks Notebook[1]. The `sourceOptions` represents a Map variable within that context. It's safe to assume that Databricks users are familiar with this setup.

[1] For more information, you can refer to: https://docs.databricks.com/en/notebooks/index.html

Reply:
@diana-macrometa This connector is used from a spark code users write on databricks platform. So each step below is decribing how someone should initialize the macrometa databricks connector within their spark source. There is no separate file or anything, users just have to use below lines of code inside their spark scripts to use our connector.
Simply think of below steps as explaining few lines of code users have to add into their source code.